src/languages/en_US/strings.json

```diff
@@ -1903,5 +1903,51 @@
   "SETTINGS_VERIFY_ROLE_DESCRIPTION": "⚙️ Manage your verify role.",
   "SETTINGS_EVENTS_DESCRIPTION": "⚙️ Manage your events settings.",
   "TOPIC_CHANGED": "Topic changed",
-  "SETTINGS_VERIFY_INTERNAL_DESCRIPTION": "⚙️ Manage your verify interal settings."
+  "SETTINGS_VERIFY_INTERNAL_DESCRIPTION": "⚙️ Manage your verify interal settings.",
+  "EMOJIS_DESCRIPTION": "⚙️ Manage your emojis.",
```

Comment:
```suggestion
```

Reply:
?

Reply:
Aren't we going to remove this command?

Reply:
Ill merge it for now since the command still exists and we dont want users seeing errors but if the comman dis removed we can remove this